index.html

```diff
@@ -2033,16 +2033,20 @@ <h4>Listing</h4>
                         
                     </section>
 
-                    <section id="validation" class="normative">
+                    <section id="exploration-directory-api-things-validation" class="normative">
```

Comment:
Modified left-over generic ID from when validation section was outside of TDD's API.